### PR TITLE
Treatment tooltip

### PR DESCRIPTION
**Pull request by pflopez**
- Adds tooltip on direct impacts map
- Adds missing title for stand chart

### CODE REVIEW

File: src/interface/src/app/treatments/map-stands-tx-result/map-stands-tx-result.component.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    .maplibregl-popup-content {
+      background-color: $color-black;
+      color: $color-white;
+      padding: 8px 12px;
+      border-radius: 8px;
+    }
+
+    // overriding tip colors for each corner
+
+    &.maplibregl-popup-anchor-right .maplibregl-popup-tip {
```
Comment (pflopez) [inline src/interface/src/app/treatments/map-stands-tx-result/map-stands-tx-result.component.scss:21]: open to suggestions on how to make all these overrides less verbose

**Comment by pflopez** [comment]
Moving it back to draft - need to check sequenced treatments

File: src/interface/src/app/treatments/map-stands-tx-result/map-stands-tx-result.component.scss (hunk 1 above)
Comment (lastminutediorama) [reply src/interface/src/app/treatments/map-stands-tx-result/map-stands-tx-result.component.scss:21]: I don't know if it's possible to centralize these styles somewhere in the future, but I imagine we will probably reuse them.

**Review by lastminutediorama — APPROVED** [review]
lgtm

**Pull request merged**